blt/src/Commands/HumsciConfigCommand.php

```diff
@@ -118,4 +118,27 @@ protected function importConfigSplit($task, $cm_core_key, $partial = FALSE) {
     $task->drush("config-import")->arg($cm_core_key)->option('partial');
   }
 
+  /**
+   * Import any missing entity form/display configs since they are ignored.
+   *
+   * @hook post-command drupal:config:import
+   */
+  public function postConfigImport() {
```

Comment:
Are you sure you want to make this assumption and not explicitly add the missing imports to an installation profile or update hook somewhere else? This seems a bit dangerous and opaque. Open for discussion.

Reply:
i thought about an update hook and that would be fine. I thought using this would eliminate the need for an update hook. instead it just always runs. the drush command to import the missing actually looks at the modules and not the config directory. so if a config lives in a module i think we can assume it should be there.

Reply:
Fair nuff, but as this a custom process and practice this will need to be documented somewhere it can get under a developer's 👃 so they are not surprised by this in the future. Perhaps outputting a message or log to the console while this is happening to let the developer know would be enough?

Reply:
I think github is delayed updating the diff, but i added a line at the top of the method:
`$this->yell('Importing missing configs!');` so you really know whats happeneing. lol

Reply:
WAT!?!? 
I couldn't hear you.

Ok, I'm going to be a bit picky. Instead of calling them missing explain how they became missing with a message like
'Importing new configuration items that don't exist in the database but are ignored in config.ignore'

Then later on do some more yelling and output the name of each. This should be helpful for debugging.

Reply:
Ah ok. LG2M.